docker-compose.yml

```diff
@@ -10,12 +10,20 @@ services:
       - "${PWD}/config/prometheus:/etc/prometheus"
     depends_on: 
       - alertmanager
+    command:
+      - '--config.file=/etc/prometheus/prometheus.yml'
+      - '--storage.tsdb.path=/prometheus'
+      - '--web.console.libraries=/etc/prometheus/console_libraries'
+      - '--web.console.templates=/etc/prometheus/consoles'
+      - '--web.enable-lifecycle'
```

Comment:
One very small thing: Maybe add a comment linking to where we got these args from, and making it clear which one is something we've added?